### DISCUSSION TITLE
Improved GitHub Pages support

### DISCUSSION THREAD

**Discussion by codesue** [maximevaillancourt/digital-garden-jekyll-template#144] [General]
It seems GitHub Pages fully supports this template now if you build and deploy the site using an Action instead of deploying from a branch. Here's the [blog post](https://github.blog/changelog/2022-07-27-github-pages-custom-github-actions-workflows-beta/) explaining how to set this up. I was able to build and deploy my site using GitHub's default Jekyll workflow.

**Comment by maximevaillancourt** [comment]
Exciting news! Thank you for reporting this, I appreciate it. I added a note in the README linking to this discussion.

**Comment by irenebosque** [comment]
Thanks for sharing this info @codesue 🙌!  I have just enabled the GitHub action to deploy my site with GitHub Pages and the custom plugin for bi-directional links works perfectly! 

For anyone else wondering how to do this: Go to Settings --> Code and automation --> Pages --> Source --> GitHub Actions, then the Jekyll workflow was selected by default.

**Comment by omarcostahamido** [comment]
Thanks @codesue @maximevaillancourt and @irenebosque for your inputs here.
I just opened #176 and the PR #177 to make it even easier.
All the best,
OCH
